cmds/tui.c

```diff
@@ -287,6 +344,36 @@ static struct display_field graph_field_total = {
 	.list = LIST_HEAD_INIT(graph_field_total.list),
 };
 
+static struct display_field graph_field_total_avg = {
+	.id = GRAPH_F_TOTAL_AVG,
+	.name = "total-avg",
+	.alias = "tot-avg",
```

Comment:
Aliases can be short like 'tavg'.

Reply:
I will apply other fields with the same rules.